content/tutorials/single-page-tutorials/setup_iota-js_ for_development.md

```diff
@@ -0,0 +1,190 @@
+node create-mnemonic.js
+```
+Now you can copy the seed phrase and paste it into the previously created `.env` file.
+
+***
+
+Your `package.json` file should look something like this:
+
+```yaml
+{
+  "name": "iota-js-setup",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "@iota/iota.js": "^1.9.0-stardust.25",
+    "bip39": "^3.0.4",
+    "dotenv": "^16.0.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0"
+}
+```
+***
+
+
+## Create an address
+
+Create a new file `create-address.js` and add the following code:
+
+
+```javascript
+const { Converter } = require("@iota/util.js");
+const { Bip32Path, Bip39 } = require("@iota/crypto.js");
+const {
+  Bech32Helper,
+  Ed25519Address,
+  Ed25519Seed,
+  ED25519_ADDRESS_TYPE,
+  generateBip44Address,
+  IOTA_BIP44_BASE_PATH,
+  SingleNodeClient,
+} = require("@iota/iota.js");
+
+require("dotenv").config({ path: "./.env" });
+
+const API_ENDPOINT = "https://api.testnet.shimmer.network";
+
+async function run() {
+  const client = new SingleNodeClient(API_ENDPOINT);
+
+  const info = await client.info();
+
+  console.log("Base");
+
+  // Generate the seed from the Mnemonic
+  const baseSeed = Ed25519Seed.fromMnemonic(process.env.MNEMONIC);
+  console.log("\tSeed", Converter.bytesToHex(baseSeed.toBytes()));
+
+  // Generate the next addresses for your account.
+  console.log();
+  console.log("Generated Addresses...");
+  const addressGeneratorAccountState = {
+    accountIndex: 0,
+    addressIndex: 0,
+    isInternal: false,
+  };
+  const path = generateBip44Address(addressGeneratorAccountState);
+  const addressSeed = baseSeed.generateSeedFromPath(new Bip32Path(path));
+  const addressKeyPair = addressSeed.keyPair();
+  const indexEd25519Address = new Ed25519Address(addressKeyPair.publicKey);
+  const indexPublicKeyAddress = indexEd25519Address.toAddress();
+
+  console.log(
+    "\tAddress (Bech32)",
+    Bech32Helper.toBech32(
+      ED25519_ADDRESS_TYPE,
+      indexPublicKeyAddress,
+      info.protocol.bech32Hrp
+    )
+  );
+}
+
+run()
+  .then(() => console.log("Done"))
+  .catch((err) => console.error(err));
+
+```
+
+If you run it, it will get your mnemonic from the `.env`file, create a seed from it and generate a new address. Just run the following command and you will see your testnet shimmer address in your console!
+
+```bash
+node create-address.js
+```
+
+## Request tokens via faucet website
+
+Now you can go to the [Shimmer Testnet Faucet Website](https://faucet.testnet.shimmer.network/), paste in your newly created address and request funds.
+
+:::note
+
+The 'Request' button will become clickable as soon as a valid Shimmer address is recognized.
+
+:::
+![Shimmer Testnet Faucet](./images/shimmer-testnet-faucet.png)
+
+***
+
+Congratulations, you are now ready to explore other exciting features of the Shimmer network, such as native tokens and NFT's.
```

Comment:
This seems like a nice place to add CTAs to the other tutorials once they're published.